package.json

```diff
@@ -1,14 +1,12 @@
 {
   "name": "@cortexapps/backstage-plugin",
   "version": "1.2.9",
```

Comment:
i would bump the lib version here, so we can test that the new published package will still work as expected

Reply:
Re: version: Sounds good!  Do we manually publish these packages, or are we luck that CI publish for us when it sees a version change?

Reply:
CI's got us 😄